js/controllers/modal/ModalHandler.js

```diff
@@ -47,7 +47,7 @@
 				titleIcon: string }} */ (this.mergeOptions(internalOptions));
 
 		// Attach content to the modal
-		$handledElement.html(this.modalBuild().html());
+		$handledElement.html(this.modalBuild()[0].outerHTML);
```

Comment:
That would have been the smart thing to check. No, I went and just generated a big diff of js files. :)